src/Component/SignUp/SignUp.js

```diff
@@ -0,0 +1,232 @@
+import React, { Component } from "react";
+import "./SignUp.scss";
+import "../../Styles/reset.scss";
+import "../../Styles/common.scss";
+
+class SignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      birthdaylist: [],
+      monthlist: [],
+      daylist: [],
+      nickname: "",
+      email: "",
+      gender_id: "",
+      birth_date: "",
+      id: "",
+      password: "",
+      year: "1990",
+      month: "4",
+      day: "15",
+    };
+  }
+
+  getBirthdayDate = () => {
+    fetch("http://localhost:3000/data/birthday.json", {
+      method: "GET",
+    })
```

Comment:
1. get method는 생략 가능합니다. 
2. localhost도 생략가능합니다 (포트번호가 바뀔 수 있기 때문에 번호가 바뀌는 경우 일일이 수정하셔야 하기때문에 삭제하는게 좋습니다 :) )
3. API주소는 config.js에서 변수에 담아 관리해주세요! (그래야 api주소가 바뀔 때 config.js에서 편리하게 수정할 수 있습니다) 

```suggestion
    fetch("/data/birthday.json")
```
```suggestion
    fetch(${birthdayAPI})
```

Reply:
네, 알겠습니다. GET 메서드는 생략했는데, fetch는 config에 저장을 해서 import를 했는데, 반영이 되지 않습니다. 뭐가 문제일 까요?

export const birthday = {
  api: "http://localhost:3000/data/birthday.json",
};

제가 한 방법이 잘못된 걸 까요?